files/config/default/etc/workflows/uct-process-upload.xml

```diff
@@ -18,6 +18,10 @@
               <input id="hasSlides" name="hasSlides" type="checkbox" class="configField" value="false" />
               <label for="hasSlides">Specify whether the video track has slides</label>
             </li>
+            <li>
+              <input id="doCaptioning" name="doCaptioning" type="checkbox" class="configField" value="false" />
+              <label for="doCaptioning">Captions</label>
```

Comment:
Please change the label text to "Captions with Google"
- because then the captions/transcript service we use is transparent.